guides/common/assembly_configuring-dhcp-integration.adoc

```diff
@@ -0,0 +1,31 @@
+include::modules/con_configuring-dhcp-integration.adoc[]
+
+include::modules/con_foreman-and-dhcp-configuration.adoc[leveloffset=+1]
+
+include::modules/con_dhcp-service-providers.adoc[leveloffset=+1]
+
+include::modules/proc_enabling-the-installer-managed-dhcp-service.adoc[leveloffset=+1]
+
+include::modules/con_integrating-a-remote-isc-dhcp-server.adoc[leveloffset=+1]
+
+include::modules/proc_integrating-infoblox-dhcp.adoc[leveloffset=+1]
+
+ifndef::satellite[]
+include::modules/proc_integrating-dnsmas-dhcp-by-using-the-libvirt-api.adoc[leveloffset=+1]
+endif::[]
+
+include::modules/proc_associating-the-dhcp-service-with-a-subnet.adoc[leveloffset=+1]
+
+include::modules/proc_disabling-dhcp-for-integration.adoc[leveloffset=+1]
+
+include::modules/proc_troubleshooting-dhcp-problems.adoc[leveloffset=+1]
+
+ifndef::satellite[]
+[appendix]
+include::modules/ref_dhcp-isc-settings.adoc[leveloffset=+1]
+
+[appendix]
+include::modules/ref_dhcp-options-for-network-configuration.adoc[leveloffset=+1]
```

Comment:
Possible future improvement: similar to above I think these this file has become obsolete. If we document `--foreman-proxy-dhcp-interface` as an optional parameter in "Enabling the installer-managed DHCP service" I think it no longer makes sense to have this.